nilearn/datasets/struct.py

```diff
@@ -451,15 +451,23 @@ def fetch_oasis_vbm(n_subjects=None, dartel_version=True, data_dir=None,
 
 def fetch_surf_fsaverage(mesh='fsaverage5', data_dir=None):
     """Download a Freesurfer fsaverage surface.
+    File names are subject to change and only attribute names
+    are guaranteed to be stable across nilearn versions.
     See [1]_.
 
     Parameters
     ----------
     mesh : str, optional
         Which mesh to fetch. Default='fsaverage5'.
 
+        - 'fsaverage3': the low-resolution fsaverage3 mesh (642 nodes)
+        - 'fsaverage4': the low-resolution fsaverage4 mesh (2562 nodes)
         - 'fsaverage5': the low-resolution fsaverage5 mesh (10242 nodes)
-        - 'fsaverage5_sphere': the low-resolution fsaverage5 spheres (10242 nodes)
+        - 'fsaverage5_sphere': deprecated;
+            since v0.7.2, sphere coordinates are directly accessible from
+            fsaverage5.sphere_{left, right}
```

Comment:
I would rather say that this option is deprecated and will raise an error in release 0.9.0 and explain the new way of getting the spheres